bgpd/bgp_vty.c

```diff
@@ -9604,27 +9604,7 @@ static void bgp_show_neighbor_graceful_restart_capability_per_afi_safi(
 			}
 
 			if (!use_json)
-				vty_out(vty, "     End-of-RIB Received   : ");
-
-			if (CHECK_FLAG(peer->af_sflags[afi][safi],
-				       PEER_STATUS_EOR_RECEIVED)) {
-				if (use_json)
-					json_object_boolean_true_add(
-						json_endofrib_status,
-						"endOfRibRecv");
-				else
-					vty_out(vty, "Yes\n");
-			} else {
-				if (use_json)
-					json_object_boolean_false_add(
-						json_endofrib_status,
-						"endOfRibRecv");
-				else
-					vty_out(vty, "No\n");
-			}
-
-			if (!use_json)
-				vty_out(vty, "     End-of-RIB Send       : ");
+				vty_out(vty, "      End-of-RIB send: ");
```

Comment:
I think End-of-RIB Sent is better than End-of-RIB Send in this context

Reply:
We have same with “Send” in the output. It’s not just visible in this given output. How should we make them unified? Which one to change: the old one or the new one?

Reply:
I meant we already have:
```
End-of-RIB send: IPv4 Unicast
End-of-RIB received: IPv4 Unicast
```
and here I did the same format.

Reply:
Yes. I see that. 

End-of-RIB Received is correct and End-of-RIB Sent is right thing.

Reply:
Older one "End-of-RIB send: IPv4 Unicast" is printed if for a given afi/safi PEER_STATUS_EOR_SEND is sent. This flag determines if sending EOR is enabled/disabled. So here End-of-RIB send make sense and other places where it talks about state of EOR we can keep it as End-of-RIB sent *.